streams/reader_from_stream_reader.ts

```diff
@@ -20,25 +19,10 @@ import type { Reader } from "../io/types.ts";
  * await copy(reader, file);
  * ```
  *
- * @deprecated (will be removed after 1.0.0) Use {@linkcode ReadableStreamDefaultReader} directly.
+ * @deprecated (will be removed in 0.220.0) Import from {@link https://deno.land/std/io/reader_from_stream_reader.ts} instead.
```

Comment:
I'd prefer to keep the original removal version

Reply:
Do you mean deprecate the `io` version? Since `readerFromStreamReader` is the inverse of `toReadableStream` it make sense to have in `io` imo.

Reply:
My suggestion is un-deprecating `readerFromStreamReader()` in `std/io` (as suggested in this PR) and keep (deprecated) `readerFromStreamReader()` in `std/streams` until v1.0 instead of removing them in `0.220.0`

> TBH - I'd vote to have the names of both of these match a little more closely. It's not obvious by the names they are related.

What is your suggestion? BTW this API has been here for relatively long time. I think randomly changing it to a different name generally has a negative effect to the ecosystem.

Reply:
> My suggestion is un-deprecating readerFromStreamReader() in std/io (as suggested in this PR) and keep (deprecated) readerFromStreamReader() in std/streams until v1.0 instead of removing them in 0.220.0

Thanks for clarifying. That makes sense to me.

> What is your suggestion? BTW this API has been here for relatively long time. I think randomly changing it to a different name generally has a negative effect to the ecosystem.

It's probably better to keep the names as is, rather then changing or introducing an alternative, etc. Maybe since `readerFromStreamReader` is being added back to the `io` namespace alongside `toReadableStream` it will be more obvious. I guess the difference in terms `StreamReader` and `ReadableStream` doesn't immediately say, these are 2 are related and opposite to each other. Naming is hard 😂.

Reply:
Removal version adjusted to v1.